docs/greptimecloud/integrations/fluent-bit.md

```diff
@@ -0,0 +1,67 @@
+---
+keywords: [Fluent Bit, GreptimeCloud, metrics ingestion, logs ingestion, data pipeline]
+description: Guide for using Fluent Bit with GreptimeCloud, including configuration for metrics and logs ingestion, and running Fluent Bit with a sample configuration.
+---
+
+
+# Fluent Bit
+
+Fluent Bit is a lightweight and fast log processor and forwarder that can collect, parse, filter, and forward logs and metrics. Fluent Bit is part of the Fluentd project ecosystem and is written in C language. It is designed to be memory-efficient and performant, making it suitable for use in resource-constrained environments.
+
+## Prometheus Remote Write
+
+Fluent Bit can be configured to send metrics to GreptimeCloud using the Prometheus Remote Write protocol. This allows you to collect metrics from various sources and send them to GreptimeCloud for storage, analysis, and visualization.
+
+```
+[OUTPUT]
+    Name                 prometheus_remote_write
+    Match                internal_metrics
+    Host                 <host>
+    Port                 80
+    Uri                  /v1/prometheus/write?db=public
+    Tls                  Off
+    http_user            <username>
+    http_passwd          <password>
+```
+
+In this example, the `prometheus_remote_write` output plugin is used to send metrics to GreptimeCloud. For more information, and extra options, refer to the [Prometheus Remote Write](https://docs.greptime.com/user-guide/integrations/prometheus) guide.
+
+## OpenTelemetry
+
+Fluent Bit can be configured to send logs and metrics to GreptimeCloud using the OpenTelemetry protocol. This allows you to collect logs and metrics from various sources and send them to GreptimeCloud for storage, analysis, and visualization.
+
+```
+# Only for metrics
+[OUTPUT]
+    Name                 opentelemetry
+    Alias                opentelemetry_metrics
+    Match                *_metrics
+    Host                 <host>
+    Port                 4000
+    Metrics_uri          /v1/otlp/v1/metrics
+    Logs_uri             /v1/otlp/v1/logs
+    Traces_uri           /v1/otlp/v1/traces
+    Log_response_payload True
+    Tls                  Off
+    Tls.verify           Off
+    Header Authorization "Basic <token> if any"
+
+# Only for logs
+[OUTPUT]
+    Name                 opentelemetry
+    Alias                opentelemetry_logs
+    Match                *_logs
+    Host                 <host>
+    Port                 80
+    Metrics_uri          /v1/otlp/v1/metrics
+    Logs_uri             /v1/otlp/v1/logs
+    Traces_uri           /v1/otlp/v1/traces
+    Log_response_payload True
+    Tls                  Off
+    Tls.verify           Off
+    Header Authorization "Basic <token> if any"
+    Header X-Greptime-Log-Table-Name "log_table"
+    Header X-Greptime-Log-Pipeline-Name "pipeline_name"
```

Comment:
The code examples in cloud integration docs must be carefully organized, especially the placeholders like `<host>`. The port 80 is incorrect. Here we cannot use token like this. And the default `pipeline_name` is not recommended either.